lib/twemoji.rb

```diff
@@ -206,12 +204,22 @@ def self.filter_emojis(content)
     # @return [String] Emoji image tag generated by name and options.
     # @private
     def self.img_tag(name)
-      unicode = render_unicode(name)
-      if options[:img_attr]
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' #{options[:img_attr]} src='#{emoji_url(name)}'>"
-      else
-        "<img class='#{options[:class_name]}' draggable='false' title='#{name}' alt='#{unicode}' src='#{emoji_url(name)}'>"
+      # set default attributes and override from img_attr
+      attributes = {
+        draggable: 'false',
+        title:     name,
+        alt:       render_unicode(name),
+        src:       emoji_url(name)
```

Comment:
I am thinking if all the attrs above should be overidable? For example.. `src` would you ever want to override that?